src/khoj/routers/helpers.py

```diff
@@ -231,34 +231,39 @@ def get_next_url(request: Request) -> str:
     return urljoin(str(request.base_url).rstrip("/"), next_path)
 
 
-def get_conversation_command(query: str, any_references: bool = False) -> ConversationCommand:
+def get_conversation_commands(query: str) -> tuple[list[ConversationCommand], bool]:
+    # Intialize variables
+    commands: list[ConversationCommand] = []
+    is_automated_task = False
+
+    # Automated Task can be mixed with other conversation commands
+    if query.startswith("/automated_task"):
+        is_automated_task = True
+        query = query.replace("/automated_task", "").lstrip()
```

Comment:
Can we move this logic of removing the `automated_task` prefix and creating the `bool` outside of this function? I'd like this function to remain a source of truth and protected from higher level business logic.

Reply:
Makes sense. Extracted the logic to handle the `automated_task` conversation command in chat API endpoint and reverted most of the `get_conversation_command` function to previous state